jwql/website/apps/jwql/data_containers.py

```diff
@@ -732,10 +737,11 @@ def get_proposal_info(filepaths):
             thumbnail = '/'.join(thumbnail.split('/')[-2:])
         thumbnail_paths.append(thumbnail)
 
-        fits_search_filepath = os.path.join(
-            FILESYSTEM_DIR, 'jw{}'.format(proposal), 'jw{}*.fits'.format(proposal)
-        )
-        num_files.append(len(glob.glob(fits_search_filepath)))
+        fits_search_filepath_proprietary = os.path.join(PROPRIETARY_FILESYSTEM_DIR, 'jw{}'.format(proposal), '*', 'jw{}*.fits'.format(proposal))
+        fits_search_filepath_public = os.path.join(PUBLIC_FILESYSTEM_DIR, 'jw{}'.format(proposal), '*', 'jw{}*.fits'.format(proposal))
+        fits_files = glob.glob(fits_search_filepath_proprietary)
+        fits_files.extend(glob.glob(fits_search_filepath_public))
+        num_files.append(len(fits_files))
```

Comment:
Will there ever be files that appear in both public and proprietary filesystems at the same time?